File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.java
class Solution {
    public String solution(String my_string) {
        String answer = "";
        
        for(int i = 0; i < my_string.length(); i++){
            char c = my_string.charAt(i);
            answer += c >= 97 ? "" + (char) (c - 32) : "" + (char) (c + 32);
        }
        
        return answer;
    }
}

File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

char* solution(const char* my_string) {
    size_t len = strlen(my_string);
    char* answer = (char*)malloc((len + 1) * sizeof(char));
    
    for(int i = 0; i < len; i++){
        if(my_string[i] >= 97){
            answer[i] = my_string[i] - 32;
        }else {
            answer[i] = my_string[i] + 32;
        }
    }
    
    answer[len] = '\0';
    
    return answer;
}